new_platforms/vsextension/ProjectTemplates/oeenclave/MyEnclave.vcxproj

```diff
@@ -123,6 +153,18 @@
     <OutDir>$(SolutionDir)bin\$(Platform)\$(Configuration)\</OutDir>
     <IntDir>$(SolutionDir)obj\$(Platform)\$(TargetName)\$(Configuration)\</IntDir>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'">
+    <GenerateManifest>false</GenerateManifest>
+    <LinkIncremental>false</LinkIncremental>
+    <IncludePath>$(NoInherit)</IncludePath>
+    <LibraryPath>$(NoInherit)</LibraryPath>
+    <OutDir>$(SolutionDir)bin\$(Platform)\$(Configuration)\</OutDir>
+    <IntDir>$(SolutionDir)obj\$(Platform)\$(TargetName)\$(Configuration)\</IntDir>
+    <NMakeBuildCommandLine>%WINDIR%\Sysnative\bash -c "make -C optee -f linux_gcc.mak"</NMakeBuildCommandLine>
```

Comment:
Is the current working directory of `bash` and thus `make` guaranteed to be such that the `optee` directory is underneath? If one runs `msbuild.exe` from the command line as part of an automated build, does this command behave the same way?

Perhaps using the full absolute path to `optee` using `$(ProjectDir)` may be safest.

Reply:
Doesn't work because $(ProjectDir) has \ not / in the path.   At least within VS, it builds fine as is.

Reply:
Verified that msbuild.exe from the command line works fine